providers/openlineage/src/airflow/providers/openlineage/utils/sql.py

```diff
@@ -157,6 +163,18 @@ def create_information_schema_query(
     sqlalchemy_engine: Engine | None = None,
 ) -> str:
     """Create query for getting table schemas from information schema."""
+    if (
+        Column is None
+        or MetaData is None
+        or Table is None
+        or and_ is None
+        or or_ is None
+        or union_all is None
+    ):
+        raise AirflowOptionalProviderFeatureException(
+            "sqlalchemy is required for SQL schema query generation. "
+            "Install it with: pip install 'apache-airflow-providers-openlineage[sqlalchemy]'"
+        )
```

Comment:
```suggestion
try:
    from sqlalchemy import Column, MetaData, Table, and_, or_, union_all
except ImportError:
    raise AirflowOptionalProviderFeatureException(
        "sqlalchemy is required for SQL schema query generation. "
        "Install it with: pip install 'apache-airflow-providers-openlineage[sqlalchemy]'"
    )
```
We need this import in only two places, we can probably go with local imports? Both are fine for me, just nit.

Reply:
this is better.

Reply:
changed it